ui/src/api.ts

```diff
@@ -78,6 +78,57 @@ export async function deleteAnnotation(arxivId: string, id: number) {
   return await axios.delete(`/api/v0/papers/arxiv:${arxivId}/annotation/${id}`);
 }
 
+export async function addLibraryEntry(paperId: string, paperTitle: string) {
+  const s2ApiUrl = `https://www.semanticscholar.org/api/1/library/entries`;
+  const tags: string[] = [];
+  const response = await axios.post(
+      s2ApiUrl,
+      {
+        paperId,
+        paperTitle,
+        tags
+      },
+      {withCredentials: true}
+  );
+  return response.data;
+}
+
+export async function getUserLibraryInfo(userId?: number) {
```

Comment:
I initially enabled caching so we don't need to make calls to get the library every time, but only on an hour basis. But as I was developing, I actually modify those values to show what the current state is anyway. I was thinking the cache would save calls if the user opens different papers in the reader, so it would use the same cache then, but its really not a big issue. I think i'll remove it then, and then we don't need to have reference to the user id anymore!